test/it/shellutils/leadtime_test.sh

```
@@ -0,0 +1,6 @@
+LeadtimeHelp() { leadtime --help; }
+LeadtimeNoSub() { leadtime; }
+LeadtimeUnknownSub() { leadtime bogus --owner=a --repo=b; }
+LeadtimeMissingToken() { LT_GITHUB_ACCESS_TOKEN= GITHUB_TOKEN= leadtime stat --owner=acme --repo=demo; }
```

Shellcheck (0.11.0)

[warning] Remove space after = if trying to assign a value (for empty string, use var='' ... ).

(SC1007)